etc/adminhtml/system/adyen_cc.xml

```diff
@@ -77,17 +77,20 @@
                 <tooltip>Enable installments for each credit card type.</tooltip>
                 <config_path>payment/adyen_cc/enable_installments</config_path>
             </field>
-            <field id="installments" type="textarea" translate="label" sortOrder="220" showInDefault="1"
-                   showInWebsite="1" showInStore="1">
+            <field id="installments" translate="label" sortOrder="220" showInDefault="1" showInWebsite="1"
+                   showInStore="1">
                 <label>Installments</label>
                 <depends>
                     <field id="enable_installments">1</field>
                 </depends>
-                <validate>required-entry</validate>
-                <comment><![CDATA[
-                    Generate the desired installments configuration <a href="">here</a> and paste it on this field.
-                ]]></comment>
-                <config_path>payment/adyen_cc/installments_code</config_path>
+                <tooltip>Configure your installment for each credit card type: Insert the minimum amount required to
+                    make the configured installment available in the amount range column.
+                    Example: if the amount range is configured to 100 and the number of installments to 4x, the shopper
+                    will see the 4x option only if the payment total is higher or equal than 100.
```

Comment:
Just wondering if the tooltip text is too long, maybe an idea to split it up between a comment and a tooltip?
(it can also stay like this)
```suggestion
                <comment>Configure your installment for each credit card type: Insert the minimum amount required to 
                make the configured installment available in the amount range column.
                </comment>
                <tooltip>
                    Example: if the amount range is configured to 100 and the number of installments to 4x, the shopper
                    will see the 4x option only if the payment total is higher or equal than 100.
```

Reply:
Nice!